types/grideyes/index.d.ts

```diff
@@ -0,0 +1,97 @@
+// Type definitions for non-npm package grideyes 0.1
+// Project: https://github.com/StormGeo/getypes
+// Definitions by: StormGeo <https://github.com/StormGeo>
```

Comment:
an organisation is not advised to be an owner of declaration
Can be referenced though